scripts-dev/complement.sh

```diff
@@ -140,7 +140,7 @@ if [[ -n "$WORKERS" ]]; then
   export PASS_SYNAPSE_COMPLEMENT_USE_WORKERS=true
 
   # Pass through the workers defined. If none, it will be an empty string
-  export PASS_SYNAPSE_WORKER_TYPES="$SYNAPSE_WORKER_TYPES"
+  export PASS_SYNAPSE_WORKER_TYPES="$WORKER_TYPES"
```

Comment:
This way it matches with the variables being declared on the commandline and environment at the same time. Explicitness is good, redundancy is bad.